client/src/nonblocking/quic_client.rs

```diff
@@ -422,3 +426,79 @@ impl QuicClient {
         self.stats.clone()
     }
 }
+
+#[derive(Clone)]
+pub struct QuicTpuConnection {
```

Comment:
Do we need this to be clonable? Can we just wrap it in an Arc?

Reply:
This needs to be clonable for the blocking quic client, which detaches when sending.  Either way, it can be wrapped in an Arc

Reply:
This needs to be clonable for the blocking quic client, which detaches when sending.  Either way, it can be wrapped in an Arc